### PR TITLE
Adding post on new changes to notifications

### PR DESCRIPTION
**Pull request by gauntface**

### CODE REVIEW

**Comment by PaulKinlan** [comment]
Side note: We need to plonk this in /web/updates/ should literally take a minute, most of the YAML is the same.

File: _posts/2015-05-22-Notifying-you-of-changes-to-notifications.md [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+}
+
+self.addEventListener('push', function(event) {  
+  console.log('Received a push message', event);
+
+  // Since this is no payload data with the first version  
+  // of Push notifications, here we'll grab some data from  
+  // an API and use it to populate a notification  
+  event.waitUntil(  
+    fetch(API\_ENDPOINT).then(function(response) {
```
Comment (ebidel) [inline _posts/2015-05-22-Notifying-you-of-changes-to-notifications.md]: `API_ENDPOINT`?

File: _posts/2015-05-22-Notifying-you-of-changes-to-notifications.md [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+    body: body,  
+    icon: icon ? icon : 'images/touch/chrome-touch-icon-192x192.png',  
+    tag: 'simple-push-demo-notification',  
+    data: data  
+  };  
+  if (self.registration.showNotification) {  
+    self.registration.showNotification(title, notificationOptions);  
+    return;  
+  } else {  
+    new Notification(title, notificationOptions);
```
Comment (ebidel) [inline _posts/2015-05-22-Notifying-you-of-changes-to-notifications.md]: might want to add a comment why there are two paths for this.

File: _posts/2015-05-22-Notifying-you-of-changes-to-notifications.md [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+notification with the purpose of just closing a notification if it's visible.
+
+At the moment there isn't a way you can do this and nothing in the spec that 
+will allow it :( but the Chrome engineering team are aware of this use case.
+
+# Android Notifications
+
+On desktop you can create a notification with the following code:
+
+new Notification('Hello', {body: 'Yay!'});
```
Comment (ebidel) [inline _posts/2015-05-22-Notifying-you-of-changes-to-notifications.md]: code block

File: _posts/2015-05-22-Notifying-you-of-changes-to-notifications.md (hunk 1 above)
Comment (gauntface) [inline _posts/2015-05-22-Notifying-you-of-changes-to-notifications.md]: Pretty much every use of push today will perform a fetch, but what it's fetching isn't relevant.

**Pull request merged**